src/V3AstNodes.cpp

```diff
@@ -18,18 +18,53 @@
 
 #include "V3EmitCBase.h"
 #include "V3File.h"
+#include "V3Global.h"
 #include "V3Graph.h"
 #include "V3Hasher.h"
 #include "V3PartitionGraph.h"  // Just for mtask dumping
 #include "V3String.h"
 
+static void dumpJsonPtr(std::ostream& os, const std::string& name, const AstNode* const valp,
+                        const std::string& unlink_msg = "UNLINKED");
```

Comment:
I generally try to avoid static file-scope functions - suggest move this to an AstNode private function.